presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/BenchmarkRunnerConfig.java

```diff
@@ -85,4 +87,16 @@ public BenchmarkRunnerConfig setJsonEventLogFile(String jsonEventLogFile)
         this.jsonEventLogFile = Optional.ofNullable(jsonEventLogFile);
         return this;
     }
+
+    public boolean getProceedWithFailures()
+    {
+        return proceedWithFailures;
+    }
```

Comment:
This method should be called `isXXX` instead of `getXXX` since it returns a boolean. You can use IntelliJ to generate getters and setters.